### ISSUE TITLE
0 lx after setup the intergration

### ISSUE THREAD

**Issue by Feriman22** [pnbruckner/ha-illuminance#74]
I added the _Illuminance_ integration in _HA_ together with _OpenWeatherMap_, both configured (hopefully) correctly, but the current lux value is zero:

![image](https://github.com/user-attachments/assets/414488d9-b914-4576-85c0-0926e7434d47)

_OpenWeatherMap_ is also configured (with _current_ mode) and it shows the data:

![image](https://github.com/user-attachments/assets/64f6ba31-4a31-4281-90aa-4bd4da283d2f)

I selected this entity when configuring the _Illuminance_ integration:

![image](https://github.com/user-attachments/assets/ed9579c1-e69b-484d-9fed-2c5aedf8722a)

Where could the issue be?

Thanks!

**Comment by pnbruckner** [comment]
Is it night time where you are?

**Comment by Feriman22** [comment]
You're right, that was the issue. I have time to configure the HA only at night, because of kids :)

Thanks!
